.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,79 @@
+name: 💎 Feature Request
+description: Request a new feature or enhancement
+labels: ['enhancement']
+title: 'Feature Request: '
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to file an enhancement report report! Please fill this template out as completely as possible.
+        * If you are not a new feature or enhancement, but want to ask a general question, use [discussions](https://github.com/muxinc/elements/discussions) instead.
+        * Check the [docs](https://docs.mux.com/guides/player?utm_source=github-mux-player).
+        * Please check to see if there's an existing issue and replying there instead.
+        * Please give as much details as you can.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+  - type: dropdown
+    id: elements
+    attributes:
+      label: Which Mux Elements/Packages does this apply to? Select all that apply
+      multiple: true
+      options:
+        - mux-video
+        - mux-video-react
+        - mux-audio
+        - mux-audio-react
+        - mux-player
+        - mux-player-react
+        - mux-uploader
+        - mux-uploader-react
+        - playback-core
+        - common
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    attributes:
+      label: Which browsers are you using?
+      description: Currently, Mux Elements are only officially supported in new version of the browsers listed.
+      multiple: true
+      options:
+        - Chrome
+        - Safari
+        - Firefox
+        - Edge ("Edgeium")
+        - Chrome Android
+        - Safari on iOS
+        - Other (add details below)
+    validations:
+      required: true
+  - type: dropdown
```

Comment:
suggestion(non-blocking): Not sure if this is relevant for feature requests? consider removing. Or, if you can imagine a value add, minimally don't make required.